src/Components/Facility/ConsultationDoctorNotes/index.tsx

```diff
@@ -38,6 +39,9 @@ const ConsultationDoctorNotes = (props: ConsultationDoctorNotesProps) => {
   const [facilityName, setFacilityName] = useState("");
   const [patientName, setPatientName] = useState("");
   const [focused, setFocused] = useState(false);
+  const [reply_to, setReplyTo] = useState<PaitentNotesReplyModel | undefined>(
+    undefined,
+  );
```

Comment:
`useState` will infer if the type is ` | undefined` or not with `T`

```suggestion
  const [replyTo, setReplyTo] = useState<PaitentNotesReplyModel>();
```